### PR TITLE
REVAI-140: Add delete jobs endpoint to docs

### PR DESCRIPTION
**Pull request by k-weng**
Corresponding implementation PR - https://github.com/revdotcom/revaiapi/pull/89

### CODE REVIEW

File: source/endpoints/jobs.rst [hunk 1]
```diff
@@ -250,5 +250,66 @@ Code                   Description
...
                           "detail": "You have only 60 seconds remaining",
                           "current_balance": 60
-                        }    
+                        }
+====================== ===============================================================
+
+``DELETE /jobs/{id}``
+*****************
+
+Deletes a transcription job by id.
```
Comment (dkokotov) [inline source/endpoints/jobs.rst]: I would add something like "All data related to the job, such as input media and transcript, will be permanently deleted. A job can only by deleted once it's completed (either with success or failure)."

File: source/endpoints/jobs.rst [hunk 2]
```diff
@@ -250,5 +250,66 @@ Code                   Description
...
+.. code:: javascript
+
+  curl -X DELETE "https://api.rev.ai/revspeech/v1beta/jobs/{id}" -H "Authorization: Bearer <api_key>"
+
+**Parameters**
+
+====================== ===============================================================
+Name                   Description
+====================== ===============================================================
+id ``*required``       Deletes an existing transcription job by id
```
Comment (dkokotov) [inline source/endpoints/jobs.rst]: description should be something like "id of transcription job to delete"

File: source/endpoints/jobs.rst [hunk 3]
```diff
@@ -250,5 +250,66 @@ Code                   Description
...
+
+                       .. code:: javascript
+
+                        {
+                          "allowed_values": [
+                            "transcribed",
+                            "failed"
+                          ],
+                          "current_value": "in_progress",
+                          "type": "https://www.rev.ai/api/speech/v1/errors/invalid-job-state",
```
Comment (dkokotov) [inline source/endpoints/jobs.rst]: should the example JSON include `status` property?

**Review by dkokotov — COMMENTED** [review]
looks pretty good

File: source/endpoints/jobs.rst (hunk 1 above)
Comment (k-weng) [reply source/endpoints/jobs.rst]: I'll add it to the implementation PR for the OpenApi docs

**Comment by k-weng** [comment]
Moving to OpenApi documentation support so no need to update them here.

**Pull request closed without merging**